### ISSUE TITLE
Can't seem to compile

### ISSUE THREAD

**Issue by Valentiino** [avoitishin/xray-16#24]
I have the enviroment for libs set as

E:\lib\

Yet when I try to compile I get several errors like

Error   4   error C1083: Cannot open include file: 'ogg/os_types.h': No such file or directory  e:\dev\libraries\libvorbis\lib\os.h 25  1   libvorbis_static

Why isn't it even looking for the includes in my set enviroment?

**Comment by Shotgunsurgeon** [comment]
Do you have E:\ as an assigned partition/drive letter? If you don't, you're pointing it to nowhere.

**Comment by Valentiino** [comment]
Yes I do, I've tried other partitions too and restarted my computer inbetween changing the enviroment

**Comment by Shotgunsurgeon** [comment]
I've had similar problems trying to utilize resources From WinDDK7.1 (ATLconv.h, et al)
I know it may not be in the spirit of github, but could someone who's managed to compile this fork of OpenXRay upload their finished files?

**Comment by Valentiino** [comment]
Alright, got through by moving some files and fixing some paths, but now I'm getting stuck at compiling the xrenderers since it's giving me missing files for luabind.hpp even if it's pointing to the right location

And this one too, which has no pointer in code

 Error  2   error C1083: Cannot open source file: 'pch.cpp': No such file or directory  H:\git\xray-16\src\3rd party\luabind\c1xx   luabind.beta7-devel.rc4

**Comment by Valentiino** [comment]
Alright, managed to kill all the other errors by myself and now I'm left with this one more error that prevents full compiling

Error   3   error C1083: Cannot open include file: 'luabind/luabind_memory_manager_generator.h': No such file or directory  H:\git\xray-16\sdk\include\boost\preprocessor\iteration\detail\iter\forward1.hpp    48  1   xrEngine

**Comment by revolucas** [comment]
How did you fix these? Ever since the 'Restructure and Cleanup' commit I've been unable to build.

**Comment by revolucas** [comment]
View > Other Windows > Property Manager > Select a project > select Release/Win32 directory > Common > VC++ Directories > Add library and include directories

**Comment by Xottab-DUTY** [comment]
I can't compile too! To be more precise, i can't compile xrGame.
Compiler falls with error:
C:\GitHub\AVO_xray-1.6\sdk\include\boost/preprocessor/iteration/detail/iter/forward1.hpp(236): fatal error C1001: Внутренняя ошибка в файле компилятора.

What can i do?

**Comment by revolucas** [comment]
Go back to the Feb 9, 2015 commit (There is no vs project for luajit-2 in this commit so you will have to add module luajit-2 repo). I've been unable to build his fork ever since the restructure and cleanup commit. Not only was I missing libraries but my Visual Studio 2013 Premium gave errors for the luajit-2 project with invalid syntax for the make command or whatever.

**Comment by Xottab-DUTY** [comment]
I think that this won't help me: I can't compile even nitrocaster's original repo (same error).

I'll try to reinstall my VS2013.
